docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Migration from the previous Api resources
+type: Details
+---
+
+The migration is done automatically by a job, which runs during the Kyma upgrade. During the migration, an old Api object is being translated to an APIRule object, which may result in a temporary downtime of the exposed service. The original resource is deleted as a part of the migration process. The tool used by the job is described [in this document](https://github.com/kyma-project/kyma/blob/master/components/api-gateway-migrator/README.md#api-gateway-migrator).
+
+>**CAUTION:** Some Api configurations are too complex or do not meet all requirements to be automatically migrated. To make sure all your services are exposed via APIRules, [verify the outcome](#verify-automatic-migration) of the migration job and follow the [manual migration process](#manual-migration) guide if necessary. Not proceeding with the manual migration won't break the existing service exposure, but any further changes or removal of the Api resource won't affect how the service is exposed - the original configuration will be used.
+
+## Verify automatic migration
+
+List the existing Api objects after the Kyma upgrade is finished. Run this command:
+
+```shell script
+kubectl get apis --all-namespaces
+```
+
+As Api objects are removed after a successful migration, all the listed resources should be considered not migrated. Fetch logs from the job's pod to learn the reason for skipping the migration. 
+
+```shell script
+kubectl logs api-gateway-api-migrator-job -n kyma-system
+```
+
+
+If the Api was not migrated due to the invalid status or a blacklisted label, you either shouldn't or will not be able to do the migration.
+
+## Manual migration
+
+The manual migration process consists of the following steps:
+
+1. Fetch the Api resource you want to migrate:
+
+```shell script
+kubectl get api <NAME> -n <NAMESPACE> -o yaml
+```
+
+2. Create an APIRule resource based on the original Api object
+
+Here is the documentation for both custom resources, which may be useful during the migration:
+
+https://kyma-project.io/docs/1.11/components/api-gateway/#custom-resource-api-sample-custom-resource
+https://kyma-project.io/docs/1.11/components/api-gateway-v2#custom-resource-api-rule
```

Comment:
>**TIP:**

maybe?

Reply:
disambiguation: **maybe** you could use the `tip` preset here. 😆

Reply:
done